homeassistant/components/totalconnect/const.py

```diff
@@ -0,0 +1,5 @@
+"""TotalConnect constants."""
+
+DOMAIN = "totalconnect"
+CONF_PASSWORD = "password"
```

Comment:
No need to define them yourself. Instead import them from `homeassistant.const`. Same for username.

Reply:
done

Reply:
Please remove them from here then.